examples/geometry/geometry_tutorial.py

```diff
@@ -68,54 +72,132 @@
     save_as_STP,
     sweep_shape,
 )
-
-# Basic objects
 from bluemira.geometry.wire import BluemiraWire
 
 # %%[markdown]
+# ## Geometry creation (1-D)
+
+# Let's get familiar with some ways of making 1-D geometries.
+# Bluemira implements functions for the creation of:
+# # * polygons
+# # * splines
+# # * arcs
+# # * a bit of everything (check geometry.tools module for an extensive list)
```

Comment:
in future we should probably link to things like this. Its easier for a user to just click on the link. Not blocking for now and probably part of a larger discussion when we do the docs review.